Cassandra CI jobs fail: SDKMAN java 17.0.12-zulu no longer available

## Summary

All Cassandra integration test CI jobs (4-LATEST and 5-LATEST) fail during setup because `java 17.0.12-zulu` has been removed from SDKMAN's distribution.

## Error

```
Downloading: java 17.0.12-zulu
Download has failed, aborting!
Can not install java 17.0.12-zulu at this time...
make[1]: *** [Makefile:345: install-java] Error 1
```

The `install-java` Makefile target uses SDKMAN to download Java when the runner doesn't have a cached `~/.sdkman` directory. Java 11 (`11.0.24-zulu`) still installs, but `17.0.12-zulu` was removed upstream.

## Fix

Update to currently available versions: `11.0.30-zulu` and `17.0.18-zulu`.

--- a/Makefile
+++ b/Makefile
@@ -347,10 +347,10 @@ install-java:
 	echo "sdkman_auto_answer=true" >> ~/.sdkman/etc/config
 	source ~/.sdkman/bin/sdkman-init.sh;
 	echo "Installing Java versions...";
-	sdk install java 11.0.24-zulu;
-	sdk install java 17.0.12-zulu;
-	sdk default java 11.0.24-zulu;
-	sdk use java 11.0.24-zulu;
+	sdk install java 11.0.30-zulu;
+	sdk install java 17.0.18-zulu;
+	sdk default java 11.0.30-zulu;
+	sdk use java 11.0.30-zulu;
 
 .prepare-cassandra-ccm:
 	@if command -v ccm >/dev/null 2>&1 && grep CASSANDRA ${CCM_CONFIG_DIR}/ccm-type 2>/dev/null 1>&2 && grep ${CCM_CASSANDRA_VERSION} ${CCM_CONFIG_DIR}/ccm-version 2>/dev//null  1>&2; then

